src/main/java/com/iheartradio/m3u8/Constants.java

```diff
@@ -111,7 +111,7 @@
 
     public static final int[] UTF_8_BOM_BYTES = {0xEF, 0xBB, 0xBF};
     public static final char UNICODE_BOM = '\uFEFF';
-    public static final int MAX_COMPATIBILITY_VERSION = 5;
+    public static final int MAX_COMPATIBILITY_VERSION = 7;    // upstream version 0.2.7 of this m3u8 code has EXT_X_VERSION:5
```

Comment:
Would it be worthwhile to add a test case that verifies parsing an .m3u8 file using this version works as expected?

Reply:
done.  added m3u8 sample files, and unit-tests in MediaPlaylistParserStartTimeTest.java

Reply:
done. added testcase in MediaPlaylistParserStartTimeTest.java